### PR TITLE
1.29 update

### PR DESCRIPTION
**Pull request by IohannesIohannium**
Changelog:
- Added support of new Manchu cultures
- New provinces added
- New countries added

### CODE REVIEW

File: EU4toV2/Data_Files/cultureMap.txt [hunk 1]
```diff
@@ -1,1130 +1,1130 @@
-# Copyright (c) 2019 The Paradox Game Converters Project
```
Comment (Idhrendur) [inline EU4toV2/Data_Files/cultureMap.txt]: Something bad happened to this file. It needs to be fixed.

Probably the line endings got changed. How did you edit it? Notepad? Try opening it in Notepad++ and using Edit->EOL Conversion->Unix

Comment (IohannesIohannium) [reply EU4toV2/Data_Files/cultureMap.txt]: I used Notepad++, but don't worry, I can fix it in a sec

Comment (IohannesIohannium) [reply EU4toV2/Data_Files/cultureMap.txt]: Somehow it didn't work. The things I edited was to simply change all instances of `eu4 = manchu` to `eu4 = manchu_new eu4 = manchu eu4 = evenk`

**Comment by IohannesIohannium** [comment]
Maybe merging your branch would be a good idea. If there's anything conflicting, the ones that need to be fixed are culture and province mappings (which I can then update in little time)

**Comment by Idhrendur** [comment]
I've merged my changes for the province mappings, but it's caused a conflict. You might need to revert your changes and do them in a separate pr.

The culture map stuff is weird. I'll try some experiments this weekend.

If you want to resubmit your country map and flags in a separate pr, I could merge that right away, at least.

**Comment by IohannesIohannium** [comment]
At the moment, I put the stuff as a pull on your branch: https://github.com/Idhrendur/EU4toVic2/pull/3

**Comment by Idhrendur** [comment]
Okay, I've got the country changes pulled into the main repo, and I took your culture mappings and submitted them on a different PR (waiting for the testers to take a look).

If you can submit your province mapping changes on a new PR, that'd be perfect!

**Pull request closed without merging**